tests/queries/0_stateless/02184_hash_functions_and_ip_types.reference

```diff
@@ -0,0 +1,43 @@
+Row 1:
+──────
+ipv4:                                        1.2.3.4
```

Comment:
BLAKE, MD*() and SHA*() are missing. Intended?

Reply:
Not intended. 
I've used `select * from system.functions where name ilike '%hash%';` to find hash functions and it obviously was not enough.